docs/content/latest/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes.md

```diff
@@ -142,12 +96,29 @@ You install Yugabyte Platform on a Kubernetes cluster as follows:
     helm install yw-test yugabytedb/yugaware --version 2.3.3 -n yb-platform --wait --set tls.sslProtocols="TLSv1.2"
     ```
 
+6. Check the services:
```

Comment:
can you add example output that shows the load balancer IP to choose for the next step.